releasenotes/notes/bump-ceph-to-19-2-1-878ae64bf5420893.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Bump Ceph to v19.2.1
```

Comment:
Should this be `upgrade` rather than `features`? Also add missing dot at end of line.

Reply:
I think you're right, I keep forgetting the meaning of each reno category.

Reply:
Can you still add the missing dot? 😄